tutorials/hana-clients-hdbsql/hana-clients-hdbsql.md

```diff
@@ -48,13 +48,64 @@ This step demonstrates how to connect to a SAP HANA instance using [HDBSQL](http
 
         > ![screenshot showing the allowlist](allowlist.png)
 
+
         >---
 
         >The SAP HANA Cloud, HANA database trial instance will be automatically stopped overnight.   That means you need to restart your instance before working with it each new day.
 
         >---
 
-        > If you are on a Linux or Mac machine and the hdbsql connection fails with the error message below, it indicates that the client could not locate a trust store in the default location.  
+        > Connections to a HANA Cloud instance must use encryption.  The default encryption library on Windows is mscrypto and on Linux and macOS it is OpenSSL.  The following example demonstrates how one could use the SAP provided conmmoncrypto library instead of the default encryption library.  Note, this requires that the SAP Client be downloaded from the SAP Software Downloads site which includes the SAP Common Crypto library (libsapcrypto).  Note that the environment variables can also be set by running source hdbclienv.sh or hdbclienv.bat.
```

Comment:
"Note, this requires that the SAP Client be downloaded from the SAP Software..."
Isn't there a verb missing?